index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tic Tac Toe</title>
+    <link href="style.css" rel="stylesheet">
+  </head>
+  <body onload="startGame();">
+    <h1>Tic Tac Toe</h1>
+    <div id="message">
+    </div>
+    <table border="1">
+      <tr>
+        <td id="s1" class="square" onClick="nextMove(this);"></td>
+        <td id="s2" class="square" onClick="nextMove(this);"></td>
+        <td id="s3" class="square" onClick="nextMove(this);"></td>
+      </tr>
+      <tr>
+        <td id="s4" class="square" onClick="nextMove(this);"></td>
+        <td id="s5" class="square" onClick="nextMove(this);"></td>
+        <td id="s6" class="square" onClick="nextMove(this);"></td>
+      </tr>
+       <tr>
+        <td id="s7" class="square" onClick="nextMove(this);"></td>
+        <td id="s8" class="square" onClick="nextMove(this);"></td>
+        <td id="s9" class="square" onClick="nextMove(this);"></td>
```

Comment:
These in-line javascript calls (`onClick="nextMove(this);`), will move into your javascript file.